internal/daemon/controller/handlers/targets/credentials.go

```diff
@@ -162,13 +162,27 @@ func staticToWorkerCredential(ctx context.Context, cred credential.Static) (sess
 			},
 		}
 
+	case *credstatic.KvCredential:
+		secrets := structpb.Struct{}
+		err := json.Unmarshal(c.GetSecrets(), &secrets)
```

Comment:
Should this be using `jsonpb` since this is a protobuf type?

Reply:
I read the package doc for jsonpb & I think you are right, because it says 
```
Do not rely on the default behavior of the standard encoding/json package when called on generated message types as it does not operate correctly.
```

But it also states that jsonpb is `Deprecated` and I should use this package instead "google.golang.org/protobuf/encoding/protojson", is that cool with you?

Reply:
Yes, definitely use the new version 👍🏻.